src/admin/requests.js

```diff
@@ -1,86 +1,81 @@
 const axios = require('axios');
 const fs = require('fs');
 
-const adminurl = "http://127.0.0.1:8001";
+const adminurl = 'http://127.0.0.1:8001';
 
+/* eslint no-console: "off" */
 
 /**
  * Function called from the admin cli, to get the Relay Admin API info
  */
-let info = function () {
-    axios.get(`${adminurl}/info`).then(function(res) {
-    	console.log(res.data);
-    });
+const info = function () {
```

Comment:
Follow pattern declaration name in function scope and skip function variable declaration
`function info () {`
Same applies to other functions in this file

Reply:
Let's move this to #55